dataland-frontend/src/components/forms/CreateEUTaxonomyForFinancials.vue

```diff
@@ -1,137 +1,521 @@
 <template>
-  <Card class="col-12">
+  <Card class="col-12 page-wrapper-card">
     <template #title>Create EU Taxonomy Dataset for a Financial Company/Service</template>
     <template #content>
-      <FormKit
-        v-model="formInputsModel"
-        :actions="false"
```

Comment:
Values in % should be correctly displayed after uploading. Proposal send inputValue/100 to the backend.

Reply:
ok